src/steps/users/contact-field-equals.ts

```diff
@@ -42,23 +43,18 @@ export class ContactFieldEquals extends BaseStep implements StepInterface {
     const stepData: any = step.getData().toJavaScript();
     const email: string = stepData.email;
     const field: string = stepData.field;
-    const operator: string = stepData.operator;
+    const operator: string = stepData.operator || 'be';
     const expectedValue: string = stepData.expectedValue;
 
     try {
       apiRes = await this.client.getContactByEmail(email);
-    } catch (e) {
-      return this.error('There was a problem checking the Contact: %s', [e.toString()]);
-    }
-
-    try {
       if (!apiRes.user) {
         // If no results were found, return an error.
         return this.error('No contact found for email %s', [email]);
-      } else if (!apiRes.user.hasOwnProperty(field) && !apiRes.user.dataFields.hasOwnProperty(field)) {
```

Comment:
Hmm...  Not sure about the changes below.  My understanding is that user (contact) field data only ever shows up on the `dataFields` property (or at least that's what I'm seeing on [Iterable's API Docs](https://api.iterable.com/api/docs#users_getUser_0)

Wouldn't this prevent checking, e.g. `firstName` field?

Reply:
![image](https://user-images.githubusercontent.com/54834859/72485661-577d1180-3843-11ea-929a-f062f174b519.png)
Upon checking again. I did successfully remove email from the rest of the fields but its still being included in dataFields. So yeah I should revert back dataFields checking but remove other user field checking.

Reply:
👍